boards/arm/nucleo_f746zg/doc/nucleof746zg.rst

```diff
@@ -0,0 +1,189 @@
+.. _nucleo_f746zg_board:
+
+ST Nucleo F746ZG
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards provide an affordable and flexible way for users to try out new concepts and build prototypes by choosing from the various combinations of performance and power consumption features, provided by the STM32 microcontroller. For the compatible boards, the SMPS significantly reduces power consumption in Run mode.
+
+The ST Zio connector, which extends the Arduino™ Uno V3 connectivity, and the ST morpho headers provide an easy means of expanding the functionality of the Nucleo open development platform with a wide choice of specialized shields.
+
+The STM32 Nucleo-144 board does not require any separate probe as it integrates the ST-LINK/V2-1 debugger/programmer.
+
+The STM32 Nucleo-144 board comes with the STM32 comprehensive free software libraries and examples available with the STM32Cube MCU Package.
+
+Key Features
+
+    STM32 microcontroller in LQFP144 package
```

Comment:
All these list items need a leading ``-``:

    - STM32 microcontroller in LQFP144 package
    - External SMPS ...